kernelci/test.py

```diff
@@ -174,5 +175,6 @@ def _get_compression(url):
 
     params.update(plan_config.params)
     params.update(target.params)
+    params.update({'device_id': device_id})
```

Comment:
Or simply `params['device_id'] = device_id` since it's a single value.

Reply:
Actually, this is better:
```suggestion
    if device_id:
        params['device_id'] = device_id
```
This way, if there is already a `device_id` set by `target.params` or something else then it only get overridden if there was also a `device_id` provided explicitly.  Hard-coding a `device_id` is actually useful and part of #1187 to only run Chrome OS tests on particular devices.  I've tested the change above by hand locally.